### DISCUSSION TITLE
Please allow the details/history split be an option

### DISCUSSION THREAD

**Discussion by ltjessem** [home-assistant/frontend#7555] [Other feature requests]
Tried to bring this up in the original PR but apparently discussion after the fact is not kosher. Created an issue [here](https://github.com/home-assistant/frontend/issues/7554) with nothing other than a "no". Apologies as I was not aware of the "discussion" area. 
Could this change not be an option so the user can decide what information is relevant to him/her?

**Comment by KTibow** [comment]
This could be moved to #7515
